### DISCUSSION TITLE
Data validation [Apr 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1718] [Discord Threads]
> 🙋‍♂️ **yallouche**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@884788718894981192> Data validations currently rely on underlying datatype supported by database. Any specific validation that you are looking at?   
####    
#### Link to another record - Yes. Information is maintained in NocoDB metadata.   
####    
#### For PG error, please help us with more details. You can raise an issue here (https://github.com/nocodb/nocodb/issues/new/choose)   
####    
#### About NocoDB- currently we are working to provide power & flexibility of database features in the hands of naive spreadsheet users & to make working with database/ api's seamless for anyone on Internet, even without programming knowledge  
#  
> 🙋‍♂️ **yallouche**  
#### very interesting I see, I've encountered quite a few PG errors, I'll note them down and write an issue  
#  
> 🙋‍♂️ **yallouche**  
#### ideally we'd like something similar to the single/multi-select button but instead of defining it within nocodb it would be based on another table   
####    
#### So in the case of  typical client/order set of table, the user would be able to pick a client from the client table when adding an order. Except that it should show up in SQL query  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/961338245944320022)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
